src/app/wallet/wallet/addresslookup/addresslookup.component.ts

```diff
@@ -104,6 +104,9 @@ export class AddressLookupComponent implements OnInit {
                   success.forEach((contact) => {
                     if (this.type === 'send' || contact.address.length > 35) {
                       this.addressLookups.push(new Contact(contact.label, contact.address));
+                    } else if (this.type === 'sign' && contact.address.length < 35 && contact.owned) {
```

Comment:
Please use the address checker in wallet/shared/utils/utils.ts

Reply:
okay :+1:

Reply:
@kewde please review this PR is ready for review now